docs/products/postgresql/concepts/timescaledb.rst

```diff
@@ -0,0 +1,31 @@
+About TimescaleDB
+=================
+
+TimescaleDB is an open-source database designed to make your existing relational database scalable for time series data. `TimescaleDB <https://github.com/timescale/timescaledb>`_ is available as a PostgreSQL extension on Aiven.
+
+A time series indexes a series of data points in chronological order, usually as a sequence over regular intervals. Examples of a time series include:
+
+* The temperature of a home during a day
+* The position of a satellite during a day
+
+The data in these examples consists of a measured value (temperature or position) corresponding to the time at which the reading of the value took place. 
+
+Enable TimescaleDB on Aiven for PostgreSQL
+------------------------------------------
+
+TimescaleDB is available as an extension; you can enable it by running::
+
+     CREATE EXTENSION timescaledb CASCADE;
+
+More information about :doc:`how to install and manage extensions <../howto/manage-extensions>` is also available.
+
+TSL-licensed features
+---------------------
+
+The majority of TimescaleDB functionality is open source, however since TimescaleDB 1.2.0 some features have been restricted to the Timescale License, which explicitly prohibits them from being made available in any database-as-a-service offering. Therefore some features, such as ``time_bucket_gapfill``, are not available on the Aiven hosted TimescaleDB service.
+
+When you try using these features, you will see an error similar to the following:
+
+``ERROR 0A000 (feature_not_supported) function "<function>" is not supported under the current license "ApacheOnly"``
```

Comment:
Minor question regarding formatting: why this row is ``ERROR 0A000.....`` and not like the usual code section? 
Is It because is an error message we are showing? If so should we apply the same format every time we show error messages for consistency?

Reply:
Good catch on consistency.
On a similar note, this is a limitation - should we have a note type for these types of comments? Or agree on naming headings of paragraphs that deal with limitations in a specific way?

Reply:
I wasn't sure how to handle it. Initially I had it in a code block but it was ... with the copy feature, it looked really wrong. It is just a log line output, not a code sample. I think we should consider using literal rather than code output where it's just output, but I don't feel strongly about it. I'd switch it back to code if it would help to get things merged.